### PR TITLE
Add secondary screening plots

### PR DESCRIPTION
**Pull request by zuzg**
Prepared plots for confirmatory screening. For know with mocked data, so some adjustment may needed later on.
image
image

One remark: for the second plot we need compound ids (each line is one compound and its values by concentrations) - I am not sure if identifying just by the location on plate is sufficient.
We can wait with merging for HTS response.

### CODE REVIEW

**Comment by zuzg** [comment]
UPDATE: 
image

File: dashboard/pages/correlation/callbacks.py [hunk 1]
```diff
@@ -102,7 +108,11 @@ def on_both_files_uploaded(
 
 
 def on_visualization_stage_entry(
-    current_stage: int, stored_uuid: str, file_storage: FileStorage
+    current_stage: int,
+    concentration_value: int,
```
Comment (Tremirre) [inline dashboard/pages/correlation/callbacks.py:112]: new params missing in docstring

**Review by Tremirre — APPROVED** [review]
nice plots! 
one docstring is missing params, other than that the code is good to go

**Comment by AndrzejKaj** [comment]
Everything looks good, I will consider legend placement since it's for both plots (?) and now it looks more like only for the left one.

**Review by AndrzejKaj — APPROVED** [review]

**Comment by zuzg** [comment]
It is only for the left one :D with regular data we will have a lot more lines on the right one, so that shouldn't be an issue

**Pull request merged**